src/components/loginRedirect/LoginRedirect.js

```diff
@@ -0,0 +1,270 @@
+
+  return (
+    <View style={styles.topContainer}>
+      <View style={styles.shadowBackground}>
+        <Section>
+          <View>
+            <View style={styles.imageContainer}>
+              <GooddollarImage />
+            </View>
+            <View style={styles.container}>
+              <Text style={styles.vendorName}>{urlDetails.vendorName}</Text>
+              <View style={styles.detailsView}>
+                <View>
+                  <Text style={{ fontSize: 16 }}>Website</Text>
+                  <Text style={{ fontSize: 10 }}>{urlDetails?.vendorURL}</Text>
+                </View>
+                <View>
+                  <Text style={{ fontSize: 16 }}>Wallet</Text>
+                  <Text style={{ fontSize: 10 }}>{urlDetails?.vendorAddress?.slice(0, 12)}...</Text>
+                </View>
+              </View>
+              <Text style={styles.boldText}>is requesting to view the following information:</Text>
+              <View style={styles.infoView}>
+                <Text style={styles.labelText}>Name</Text>
+                <Text>{fullName}</Text>
+              </View>
+              <View style={styles.infoView}>
+                <Text style={styles.labelText}>Mobile</Text>
+                <Text>{mobile}</Text>
+              </View>
+              <View style={styles.infoView}>
+                <Text style={styles.labelText}>Email</Text>
+                <Text>{email}</Text>
+              </View>
+              <View style={styles.infoView}>
+                <Text style={styles.labelText}>Location</Text>
+                <Text>{country}</Text>
+              </View>
+              <View style={styles.infoView}>
+                <Text style={styles.labelText}>Wallet Address</Text>
+                <Text>{urlDetails?.vendorAddress}</Text>
+              </View>
+              <View style={styles.infoView}>
+                <Text style={styles.labelText}>GoodDollar verification status</Text>
+                {!isVendorWalletWhitelisted && (
+                  <View style={styles.unVerifiedView}>
+                    <Text style={styles.unVerifiedText}>Not Verified</Text>
+                  </View>
+                )}
+                {isVendorWalletWhitelisted && (
+                  <View style={styles.verifiedView}>
+                    <Text style={styles.verifiedText}>Verified</Text>
+                  </View>
+                )}
+              </View>
+              <View style={styles.buttonContainer}>
+                <CustomButton onPress={deny} style={styles.denyButton}>
+                  <Text style={{ color: '#00AFFF' }}>Deny</Text>
+                </CustomButton>
+                <CustomButton onPress={allow} style={styles.allowButton}>
+                  Allow
+                </CustomButton>
+              </View>
+            </View>
+          </View>
+        </Section>
+      </View>
+    </View>
+  )
+}
+
+const getStylesFromProps = ({ theme }) => {
+  return {
```

Comment:
you don't use theme
there are some colors / padding sizes pre-defined

please check other components as examples, also look to src/components/theme/styles.js